### DISCUSSION TITLE
how to set cpp indent as 4 space when save files?

### DISCUSSION THREAD

**Discussion by edte** [LunarVim/LunarVim#2800] [Q&A] [answered]
the default indent is 2 space,

![a](https://user-images.githubusercontent.com/50194671/177959638-b8dd48c8-28e5-43dc-8b79-5a8bc345d671.gif)

And how to set the number of words for automatic line wrapping? It's too early to wrap

**Comment by kylo252** [comment, marked as answer]
use [`ftplugin`](https://neovim.io/doc/user/usr_41.html#ftplugin)

```lua
-- in ~/.config/lvim/after/ftplugin/cpp.lua
vim.cmd [[setlocal tabstop=4 shiftwidth=4 textwidth=100]]
```
